modules/apps/object/object-test/src/testFunctional/tests/Object.testcase

```diff
@@ -21,6 +21,143 @@ definition {
 		}
 	}
 
+	@description = "LPS-TEST - Verify if it is possible to create a Custom Object"
+	@priority = 5
+	test CanCreateCustomObject {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 01",
+			objectName = "CustomObject01",
+			pluralLabelName = "Custom Objects 01");
+
+		ObjectAdmin.openObjectAdmin();
+
+		Refresh();
+
+		Pause(locator1 = 3000);
+
+		AssertElementPresent(
+			key_label = "Custom Object 01",
+			locator1 = "ObjectAdmin#SELECT_OBJECT_FIELD");
+	}
+
+	@description = "LPS-TEST - Verify it is possible to create a String field"
+	@priority = 5
+	test CanCreateStringFieldInCustomObject {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 01",
+			objectName = "CustomObject01",
+			pluralLabelName = "Custom Objects 01");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object 01");
+
+		ObjectAdmin.goToFieldsTab();
+
+		LexiconEntry.gotoAdd();
+
+		Pause(locator1 = 2000);
+
+		Type(
+			locator1 = "ObjectField#LABEL_INPUT",
+			value1 = "Custom String Field 01");
+
+		ObjectField.selectType(fieldType = "Text");
+
+		Pause(locator1 = 2000);
+
+		PortletEntry.save();
+
+		Refresh();
+
+		Pause(locator1 = 2000);
+
+		AssertElementPresent(
+			key_label = "Custom String Field 01",
+			locator1 = "ObjectAdmin#SELECT_OBJECT_FIELD");
+	}
+
+	@description = "LPS-TEST - Verify it is possible to add a field after the Object is published and submit entries to it"
+	@priority = 5
+	test CanAddFieldAfterObjectPublishedAndSubmitEntries {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 777",
+			objectName = "CustomObject777",
+			pluralLabelName = "Custom Objects 777");
+
+		Navigator.openURL();
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object 777");
+
+		ObjectAdmin.goToFieldsTab();
+
+		ObjectAdmin.addObjectFieldViaUI(
+			fieldLabel = "Full Name",
+			fieldType = "Text");
+
+		Refresh();
+
+		ObjectAdmin.assertObjectField(
+			fieldLabel = "Full Name",
+			fieldType = "Text");
+
+
+		ObjectAdmin.goToDetailsTab();
+
+		CreateObject.selectPanelCategoryKey(panelCategoryKey = "Control Panel > Object");
+
+		ObjectAdmin.publishObjectViaUI();
+
+		Navigator.openURL();
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object 777");
+
+		ObjectAdmin.goToFieldsTab();
+
+		ObjectAdmin.addObjectFieldViaUI(
+			fieldLabel = "Phone Number",
+			fieldType = "Long Integer");
+
+		Refresh();
+
+		ObjectAdmin.assertObjectField(
+			fieldLabel = "Phone Number",
+			fieldType = "LongInteger");
+
+		
+
+		ApplicationsMenu.gotoPortlet(
+				category = "Object",
+				panel = "Control Panel",
+				portlet = "Custom Objects 777");
+
+		LexiconEntry.gotoAdd();
+
+		WaitForPageLoad();
+
+		ObjectPortlet.typeCustomField(
+				customField = "Full Name",
+				entry = "Ronaldo Fenomeno");
+
+		ObjectPortlet.typeCustomField(
+				customField = "Phone Number",
+				entry = "5545954341434");
+
```

Comment:
These steps are not necessary as there is already a macro that adds entries. Please give a feel for the object related test files

Reply:
ok, i will try change this